.gitignore

```diff
@@ -0,0 +1,3 @@
+.idea/*
```

Comment:
you can expand the .gitignore file for all necessary cases, including some specific IDE/management configs and building files; more details there https://www.toptal.com/developers/gitignore
using this generator you can specify the main Frameworks/tools/BM of yours develop project and it will be enough to cover the main cases of .gitignore file;